examples/oauth.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+require('../init.php');
+
+\Stripe\Stripe::setApiKey(getenv('STRIPE_SECRET_KEY'));
+\Stripe\Stripe::setClientId(getenv('STRIPE_CLIENT_ID'));
+
+
+if (isset($_GET['code'])) {
+
+    $code = $_GET['code'];
+
+    try {
+        $resp = \Stripe\OAuth::token(array(
+            'grant_type' => 'authorization_code',
+            'code' => $code,
+        ));
+    } catch (\Stripe\Error\OAuth\OAuthBase $e) {
+        exit("Error: " . $e->getMessage());
+    }
+
+    $accountId = $resp->stripe_user_id;
+
+    echo "<p>Success! Account <code>$accountId</code> is connected.</p>\n";
+    echo "<p>Click <a href=\"?deauth=$accountId\">here</a> to disconnect the account.</p>\n";
+
+} elseif (isset($_GET['deauth'])) {
+    $accountId = $_GET['deauth'];
+
+    try {
+        \Stripe\OAuth::deauthorize(array(
+            'stripe_user_id' => $accountId,
+        ));
+    } catch (\Stripe\Error\OAuth\OAuthBase $e) {
+        exit("Error: " . $e->getMessage());
+    }
+
+    echo "<p>Success! Account <code>$accountId</code> is disonnected.</p>\n";
+    echo "<p>Click <a href=\"?\">here</a> to restart the OAuth flow.</p>\n";
+
+} else {
+    $url = \Stripe\OAuth::authorizeUrl(array(
```

Comment:
You're not handling the error case here for example when the customer decides not to Connect or if something wrong happens. There would be no `code` in the URL but there would be an `error` parameter that should be handled.